src/content/peerconnection/constraints/js/main.js

```diff
@@ -249,20 +249,31 @@ setInterval(function() {
 
       // search for the candidate pair
       results.forEach(function(report) {
-        if (report.type === 'candidatepair' && report.selected ||
-            report.type === 'googCandidatePair' &&
-            report.googActiveConnection === 'true') {
-          activeCandidatePair = report;
+        if (report.type === 'transport') { 
+          activeCandidatePair = results.get(report.selectedCandidatePairId); 
         }
       });
+      if (!activeCandidatePair) {
+        results.forEach(function(report) {
+          if (report.type === 'candidate-pair' && report.selected ||
```

Comment:
Can you merge these two searches and add a comment that transport.selectedCandidatePairId is spec way and candidatePair.selected is Firefox way?